src-gui/src/renderer/components/modal/feedback/FeedbackDialog.tsx

```diff
@@ -1,356 +1,222 @@
+                                            checked={input.attachDaemonLogs}
+                                            onChange={(e) =>
+                                                setInputState((prev) => ({
+                                                    ...prev,
+                                                    attachDaemonLogs:
+                                                        e.target.checked,
+                                                }))
+                                            }
+                                        />
+                                    }
+                                    label="Attach logs from the current session"
+                                />
+                            </Paper>
+                            <Tooltip title="View the logs">
+                                <Box
+                                    style={{
+                                        display: 'flex',
+                                        alignItems: 'center',
+                                        justifyContent: 'center',
+                                    }}
+                                >
+                                    <IconButton
+                                        onClick={() =>
+                                            setDaemonLogsEditorOpen(true)
+                                        }
+                                        disabled={
+                                            input.attachDaemonLogs === false
+                                        }
+                                    >
+                                        <Visibility />
+                                    </IconButton>
+                                </Box>
+                            </Tooltip>
+                        </Box>
+                    </Box>
+                    <Typography
+                        variant="caption"
+                        color="textSecondary"
+                        style={{ marginBottom: '0.5rem' }}
+                    >
+                        Your feedback will be answered in the app and can be
+                        found in the Feedback tab
+                    </Typography>
+                    <LogViewer
+                        open={daemonLogsEditorOpen}
+                        setOpen={setDaemonLogsEditorOpen}
+                        logs={logs.daemonLogs}
+                        setIsRedacted={(redact) =>
+                            setInputState((prev) => ({
+                                ...prev,
+                                isDaemonLogsRedacted: redact,
+                            }))
+                        }
+                        isRedacted={input.isDaemonLogsRedacted}
+                    />
+                </Box>
+            </DialogContent>
+            <DialogActions>
+                <Button onClick={handleClose}>Cancel</Button>
+                <PromiseInvokeButton
```

Comment:
You need to explicitly set the `requiresContext` prop to false. Otherwise users cannot send feedback if the context initializations (wallet, tor, ...) fails